tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -112,12 +120,60 @@ def validate_dut_routes_exist(duthosts, rand_one_dut_hostname, dut_dhcp_relay_da
         assert len(rtInfo["nexthops"]) > 0, "Failed to find route to DHCP server '{0}'".format(dhcp_server)
 
 
-def test_dhcp_relay_default(duthosts, rand_one_dut_hostname, ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist):
+def restart_dhcp_service(duthost):
+    duthost.shell('systemctl restart dhcp_relay')
+    duthost.shell('systemctl reset-failed dhcp_relay')
```

Comment:
please, move reset-failed before the restart dhcp_service

Reply:
reason?

Reply:
I was suspecting the dhcp is restart by the test case multiple time, however I checked and it is only once.

Reply:
i add it both before and after which is should be much safer